### PR TITLE
Add `SSL` error to FAQ

### PR DESCRIPTION
**Pull request by itscynxx**
The question's wording is a bit rough and I could only go off of a couple random messages about it from searching in the server. Part of the process of moving the faq discord channel to the wiki's faq page

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -77,3 +77,7 @@ If you want to use Northstar you'll have to purchase Titanfall 2 either from Ste
 ### Q: Where is Frontier Defense?
 
 A: Frontier Defense is currently in development and will be available in a release when it's finished.
+
+### Q: I get an error message saying something along the lines of "SSL connect error" 
+
+A: There is currently no solution to this sadly, and we aren't sure what causes it. If you get this error, please create a ticket about it in the Northstar Discord server.
```
Comment (GeckoEidechse) [inline docs/faq.md]: I'd rephrase it to something along the lines of "we are actively investigating the error but have not found a solution yet. Please reach out to us via a ticket in the Northstar Discord so that we can gather more information on the issue"

**Comment by pg9182** [comment]
I have some theories about this. Next time this happens, can you get whoever it is (if they're on linux) to run (and send me the full output of):

```bash
dig -tcp +noedns northstar.tf A
dig +tcp northstar.tf A
dig -4 +trace northstar.tf A
curl -4 -XHEAD -vvi https://northstar.tf
```

**Review by GeckoEidechse — APPROVED** [review]
Requested changes addressed, looks good now, thanks ^^

**Comment by GeckoEidechse** [comment]
> I have some theories about this. Next time this happens, can you get whoever it is (if they're on linux) to run (and send me the full output of):
> 
> ```shell
> dig -tcp +noedns northstar.tf A
> dig +tcp northstar.tf A
> dig -4 +trace northstar.tf A
> curl -4 -XHEAD -vvi https://northstar.tf
> ```

Actually depending on the expected output, I could probably add something to [FlightCore](https://github.com/R2NorthstarTools/FlightCore/) to check for it if we wanna :eyes:
